go/internal/indexerdb/dao.go

```diff
@@ -0,0 +1,17 @@
+package indexerdb
+
+type Dao struct {
+	Id                     uint32 `gorm:"primaryKey;autoIncrement"`
```

Comment:
this id should not be managed by the indexer, it should be derived from onchain data uniquely identifying this dao, like main contract address

Reply:
done